hack/generated/config/samples/microsoft.networking/v1alpha120201101_loadbalancer.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: microsoft.networking.infra.azure.com/v1alpha1api20201101
```

```suggestion
apiVersion: microsoft.network.infra.azure.com/v1alpha1api20201101
```

And the directory should be `microsoft.network` for consistency.

---

Thanks!